include/realtime_tools/realtime_publisher.h

```diff
@@ -204,8 +205,8 @@ class RealtimePublisher
   }
 
   PublisherSharedPtr publisher_;
-  volatile bool is_running_;
-  volatile bool keep_running_;
+  std::atomic_bool is_running_;
+  std::atomic_bool keep_running_;
```

Comment:
As mentioned in #50, `std::atomic_flag` is more suitable here since it's guaranteed to be implemented lock-free.

Reply:
Ah, I didn't know that - Thanks!